src/content/docs/en/hardware/requirements.mdx

```diff
@@ -5,34 +5,34 @@ description: "Minimum Requirements for Ultramarine Linux"
 
 import Alert from "../../../../components/Docs/Alert.astro";
 
-## Recommended Requirements for Standard PCs
+## Recommended Specifications for Standard PCs
 
 These are our suggestions for the most basic (daily-driveable) system:
 
 - 4GB of RAM (8GB or more recommended)
```

Comment:
```suggestion
- 4GB of RAM
```
Adding (8GBs recommended) seems redundant since its the 'most basic' suggestion, and 4GBs ram will run the OS just fine for basic tasks.

Reply:
no, we suggest more storage here, and this is important information

Reply:
Alright that's fine